apis/v1beta1/validation/httproute.go

```diff
@@ -77,6 +78,17 @@ func ValidateHTTPRouteSpec(spec *gatewayv1b1.HTTPRouteSpec, path *field.Path) fi
 	return errs
 }
 
+// validateRequestRedirectFiltersWithBackendRefs validates that RequestRedirect filters are not used with backendRefs
+func validateRequestRedirectFiltersWithBackendRefs(rule gatewayv1b1.HTTPRouteRule, path *field.Path) field.ErrorList {
+	var errs field.ErrorList
+	for _, filter := range rule.Filters {
+		if filter.RequestRedirect != nil && len(rule.BackendRefs) > 0 {
+			errs = append(errs, field.Invalid(path.Child("filters"), filter, "RequestRedirect filter is not allowed with backendRefs"))
```

Comment:
is `filter,` needed ?

Reply:
If you are asking if `filter` needed inside the `field.Invalid` function. So yeah I think we can change to `gatewayv1b1.HTTPRouteFilterRequestRedirect` .. as it will be more specific.
```
errs = append(errs, field.Invalid(path.Child("filters"), gatewayv1b1.HTTPRouteFilterRequestRedirect, "RequestRedirect filter is not allowed with backendRefs"))
```

Reply:
thanks, yeah +1 to this change